<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student_Grade_System_GUIClass</class>
 <widget class="QMainWindow" name="Student_Grade_System_GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student_Grade_System_GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>410</width>
      <height>60</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameLineEdit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Faculty:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="facultyLineEdit"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Faculty Number:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="facultyNumLineEdit"/>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Group Number:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="groupNumLineEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>        Semester 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>         Semester 2</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>224</width>
      <height>146</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLineEdit" name="sem1Grade1LineEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="sem2Grade1LineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="sem1Grade2LineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="sem2Grade2LineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="sem1Grade3LineEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="sem2Grade3LineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="sem1Grade4LineEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="sem2Grade4LineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="sem1Grade5LineEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="sem2Grade5LineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="showDetails">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>155</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Details</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>320</y>
      <width>621</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="addStudentButton">
       <property name="text">
        <string>Add Student</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveToFileButton">
       <property name="text">
        <string>Save to File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calculateButton">
       <property name="text">
        <string>Calculate Average - Group</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calculateCurrentButton">
       <property name="text">
        <string>Calculate Average - Student</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>623</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Student_Grade_System_GUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addStudentButton</sender>
   <signal>clicked()</signal>
   <receiver>Student_Grade_System_GUIClass</receiver>
   <slot>on_addStudentButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveToFileButton</sender>
   <signal>clicked()</signal>
   <receiver>Student_Grade_System_GUIClass</receiver>
   <slot>on_saveToFileButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calculateButton</sender>
   <signal>clicked()</signal>
   <receiver>Student_Grade_System_GUIClass</receiver>
   <slot>on_calculateButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_addStudentButton_clicked()</slot>
  <slot>on_saveToFileButton_clicked()</slot>
  <slot>on_calculateButton_clicked()</slot>
 </slots>
</ui>
